src/transforms/add_fields.rs

```diff
@@ -51,7 +51,11 @@ inventory::submit! {
     TransformDescription::new::<AddFieldsConfig>("add_fields")
 }
 
-impl GenerateConfig for AddFieldsConfig {}
+impl GenerateConfig for AddFieldsConfig {
+    fn generate_config() -> toml::Value {
+        toml::from_str(r#"fields.name = "TODO""#).unwrap()
```

Comment:
Is this something that needs completing? If not, I suggest changing the name above.

Reply:
changed to `field_name`